### DISCUSSION TITLE
"No indexed repository" in Code Browser for member account

### DISCUSSION THREAD

**Discussion by joaodinissf** [TabbyML/tabby#2110] [Q&A] [answered]
I would like to configure custom local repositories in Tabby server's "Code Browser" view, as described [here](https://tabby.tabbyml.com/docs/administration/context/).

I followed these steps:

1. Using my team's owner account, I set up multiple local repositories under "Integrations" > "Repository Providers":

![image](https://github.com/TabbyML/tabby/assets/6786818/da6f5797-eb8f-4452-a300-95f4326832c2)

2. When using this account, everything seems to work correctly: if I access the Code Browser at `http://${TABBY_SERVER_URL}/files`, I see the four repositories I configured, and I am able to browse the files:

![image](https://github.com/TabbyML/tabby/assets/6786818/28dd1d4d-c246-48fa-9361-18b2db266402)

3. However, when I log in with a member account from my team and access the Code Browser at the same URL, I see a message indicating that no repositories have been indexed yet:

![image](https://github.com/TabbyML/tabby/assets/6786818/49fb4251-ba07-4a98-940c-880ea4529812)

I would expect the same four repositories to be available to all users in the account.

How can I achieve this?

If it matters, I am currently on the Community plan.

**Comment by wsxiaoys** [comment, marked as answer]
Thanks for detailed bug report - this indeed is a bug. Filing https://github.com/TabbyML/tabby/pull/2111 to fix.

**Reply by wsxiaoys** [reply]
released in https://github.com/TabbyML/tabby/releases/tag/v0.11.1
